common/src/main/scala/org/neo4j/spark/util/Neo4jImplicits.scala

```diff
@@ -112,6 +139,22 @@ object Neo4jImplicits {
     }
 
     def getAttributeWithoutEntityName: Option[String] = filter.getAttribute.map(_.unquote().split('.').tail.mkString("."))
+
+    def getAttributeAndValue: Seq[Any] = {
```

Comment:
why we don't manage EqualNullSafe and And/Or?

Reply:
And/Or are handled by [this function](https://github.com/neo4j-contrib/neo4j-spark-connector/pull/377/files#diff-47b5cfa4eae29a6ccac76fcffed8b50a0851e27835665551374237611d31f76aR97-R103) called [here](https://github.com/neo4j-contrib/neo4j-spark-connector/pull/377/files#diff-7efd6a0c28a95713ded5f486215a2d67baca6402591f2ce6423ace245de612a1R249-R252). I added a comment to explain better.

Added EqualNullSafe case.